### ISSUE TITLE
3 binaries after install instead of one

### ISSUE THREAD

**Issue by adriangalilea** [ralphbean/bugwarrior#1043]
Hello,

I've noticed that after install I get 3 binaries

```
ls /Library/Frameworks/Python.framework/Versions/3.11/bin | grep bugwarrior

bugwarrior-pull
bugwarrior-uda
bugwarrior-vault
```

This seems to contradict the documentation, I tried installing from pip and from source just to see if there was something wrong with one of the installing methods.

Is this to be expected?

I'm on macOS.

**Comment by ryneeverett** [comment]
> This seems to contradict the documentation

I have a feeling this is another case of #1007. I'm guessing you're looking at the [Command Line Interface](https://bugwarrior.readthedocs.io/en/latest/manpage.html) page of the "latest" documentation but are installing the "stable" 1.8.0 release.

For context, the top-level `bugwarrior` command with subcommands has not yet been released. As of right now the top-level command is added as a fourth binary. The original three binaries are still shipped but are no longer documented. I was originally planning to ship the next release with all 4 binaries but now that it's determined the next release will be a major release it's probably best to go ahead and drop them.

**Comment by adriangalilea** [comment]
You are absolutely correct @ryneeverett However if I set "stable" on  the documentation it gives a 404 :)

No worries, I'll continue using the 3 commands for now.

**Comment by ryneeverett** [comment]
> However if I set "stable" on the documentation it gives a 404 :)

Yes, I ran into that myself and it is most annoying. I believe it only happens when you're on a page that didn't exist in the "stable" documentation.

**Comment by adriangalilea** [comment]
> > However if I set "stable" on the documentation it gives a 404 :)
> 
> Yes, I ran into that myself and it is most annoying. I believe it only happens when you're on a page that didn't exist in the "stable" documentation.

Got it, perhaps a banner on top of the dev docs would suffice in the meantime, with a link to the relevant docs :)

> WARNING: you are navigating the dev docs, but you probably installed 1.8 go here to see the docs for your version.

**Comment by ryneeverett** [comment]
On second thought, I'm not sure dropping the legacy cli makes sense for the next release. The top level bugwarrior command was added and the old cli was deprecated since the last release and it might be best to ship all 4 binaries in at least one release.
